docs/gallery/index.md

```diff
@@ -1,36 +1,26 @@
 # Gallery of Previous Hackweeks
 
-````{panels}
-:container: full-width
-:column: text-center col-6 col-lg-4
-:card: +my-2
+````{grid}
 
-![SnowExHackweek logo](https://snow.nasa.gov/sites/default/files/snowex-logo.png)
-
-```{link-button} SnowExhackweek
-:text: SnowEx Hackweek
-:type: ref
-:classes: stretched-link
+```{grid-item-card} SnowEx Hackweek 2021
+:img-top: https://snow.nasa.gov/sites/default/files/snowex-logo.png
+:link: SnowExhackweek
+:link-type: doc
+:text-align: center
 ```
 
----
-
-![ICESat-2 hackweek logo](http://www.nasa.gov/sites/default/files/thumbnails/image/icesat2-logo_0.jpg)
-
-```{link-button} ICESat-2hackweek
-:text: ICESat-2 Hackweek
-:type: ref
-:classes: stretched-link
+```{grid-item-card} ICESat-2 Hackweek 2020
```

Comment:
Should we update to 2022?

Reply:
It actually links to 2021 content only right now! We should probably just omit the date and have subsections for different events on https://guidebook.hackweek.io/gallery/SnowExhackweek.html ...